src/compiler/expression/object/PropertyAssignment.ts

```diff
@@ -58,7 +58,30 @@ export class PropertyAssignment extends PropertyAssignmentBase<ts.PropertyAssign
                 textLength: initializer.getWidth()
             }
         });
-
         return this;
     }
+
+    /**
+     * Removes this property.
+     */
+    remove() {
+        const children: Node[] = [];
+        const previousComma = this.getPreviousSiblingIfKind(SyntaxKind.CommaToken);
+        let removePrecedingSpaces = false;
+        let removeFollowingSpaces = false;
+
+        if(previousComma) {
+            children.push(previousComma);
+            removePrecedingSpaces = true;
+        }
+        children.push(this);
+        if (!previousComma) {
+            const nextComma = this.getNextSiblingIfKind(SyntaxKind.CommaToken);
+            if(nextComma) {
+                children.push(nextComma);
+                removeFollowingSpaces = true;
+            }
+         }
+        removeChildren({ children, removePrecedingSpaces, removeFollowingSpaces });
```

Comment:
Looks good, but I don't think this will work when the members are on new lines. Perhaps you could adapt this to use `removeCommaSeparatedChild` instead and make it work for newlines?